components/worktemplates/components/modal.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import styled from 'styled-components';
+
+import GenericModal from 'components/generic_modal';
+
+const Modal = styled(GenericModal)`
+    width: 960px;
```

Comment:
Should take into account tablet and mobile [screen width](https://github.com/mattermost/mattermost-webapp/blob/master/utils/constants.tsx#L1378-L1380).

Reply:
It works fine on desktop and tablet, but I'm unsure how to address the feature on mobile.  @annelieseklein is this something that's been considered?